# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/pciutils/pciutils-use-sys_io.h.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

always use <sys/io.h>

--- a/lib/i386-io-linux.h	2014-04-26 01:17:03.806539053 +0200
+++ b/lib/i386-io-linux.h	2014-04-26 01:17:24.403019954 +0200
@@ -6,11 +6,7 @@
  *	Can be freely distributed and used under the terms of the GNU GPL.
  */
 
-#ifdef __GLIBC__
 #include <sys/io.h>
-#else
-#include <asm/io.h>
-#endif
 
 static int
 intel_setup_io(struct pci_access *a UNUSED)
